### PR TITLE
/areas_cleanup for removing ownerless areas

### PR DESCRIPTION
**Pull request by MoNTE48**
Not sure if anyone else needs this. I created my fork of this good mod some time ago.

During this time, more than 10,000 areas appeared on my server. I became worried that after removing a player, all of his areas are saved. Manually checking and clearing 10,000 records is not possible. So I add chatcommand for it.
If only I need this, I'm sorry.

### CODE REVIEW

**Comment by Panquesito7** [comment]
This new command is not listed on `README.md`, it should be added there.

**Comment by MoNTE48** [comment]
Thanks for the reminder, fixed!

`  * areas_cleanup -- Removes all ownerless areas.
	Useful for cleaning after user deletion, for example using /remove_player.`

File: README.md [hunk 1]
```diff
@@ -101,6 +101,9 @@ Commands
 
   * `/area_pos2 [X,Y,Z|X Y Z]` -- Sets area position two to your position or
 	the one supplied.
+	
+  * `areas_cleanup` -- Removes all ownerless areas.
```
Comment (Panquesito7) [inline README.md]:
```suggestion
  * `/areas_cleanup` -- Removes all ownerless areas.
```
Missing `/`.

**Comment by MoNTE48** [comment]
@SmallJoker If anyone other than me is interested in this...

**Comment by BuckarooBanzay** [comment]
(maybe offtopic)

> I became worried that after removing a player, all of his areas are saved

@MoNTE48 out of curiosity: on what basis do you remove players?

**Comment by MoNTE48** [comment]
1) The DB of more than 0.5 million players has become too large, many good names are occupied
2) The player has not visited for more than 6 months. Then his datas is deleted, and his area is also freed.

**Comment by SmallJoker** [comment]
Looks good. Will squash my commit into N°2 (fa11486) and merge this PR in a few days unless there are objections.

**Pull request merged**